Vizzuality/half-earth-v3
# Feature/share aois

## Save shared AOIs to DB and retrieve them
### Description
The shared AOIs are saved to DB when the share modal opens (only if they are custom)
They will be only saved once on this service:
https://services9.arcgis.com/IkktFdUAcY3WrH25/ArcGIS/rest/services/shared_custom_aois/FeatureServer/0

TODO: FIx the edit name functionality so we can update the stored area when we change its name (will require online config)

### Testing instructions
- Create a new custom AOI
- Open the share modal and copy the share link
- Open a new window on incognito mode and paste the link
- The area data and geometry should appear

### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-204?atlOrigin=eyJpIjoiYzZkOTNmN2UwNjA5NGFlMmE3Zjc5ZGJhM2ZjMDYwMTAiLCJwIjoiaiJ9
